static/scss/profile.scss

```diff
@@ -89,6 +100,18 @@
       }
     }
 
+    .profile-row-icons {
+      text-align: right;
+      color: #898989;
+
+      &.alignment {
+        float: right;
+        position: absolute;
+        right: 0px;
+        padding-right: 37px;
+      }
+    }
```

Comment:
here I think you should be able to just set:

```scss
.profile-row-icons {
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  width: 50%;
}
```

something along those lines.